libraries/AP_NavEKF2/AP_NavEKF2.cpp

```diff
@@ -123,6 +123,7 @@ const AP_Param::GroupInfo NavEKF2::var_info[] = {
     // @Description: This enables EKF2. Enabling EKF2 only makes the maths run, it does not mean it will be used for flight control. To use it for flight control set AHRS_EKF_TYPE=2. A reboot or restart will need to be performed after changing the value of EK2_ENABLE for it to take effect.
```

Comment:
Is it not better to always warn the user ? When they change something and it does not work, it's confusing. Some people might not come to the idea that they need a reboot, and simply try to find other parameters to change, or get frustrated. If a reset is needed in some situations just warn all the time. This way it will work all the time and not only in some situations. BTW where is it documented that it is requires reboot when disabling and not on enabling ? And for other parameters ? Where are the docs, will the users find them, will they even search for it ?

I vote that is better to request a reset more than a reset less. :)